### DISCUSSION TITLE
Ckan datapusher+ dockerized

### DISCUSSION THREAD

**Discussion by FWICSS** [ckan/ckan#8117] [Q&A]
Hello everyone,
I tried to make a Dockerized CKAN with datapusher+ but I can't do it, do you have a solution because I want to recover the data from the xlsx files which does not work with datapusher
https://github.com/FWICSS/dockerized-ckan

**Comment by muhammetcetinyamac** [comment]
Hello, could you please share the error you encountered?

**Reply by FWICSS** [reply]
![CleanShot 2024-06-10 at 11 57 38](https://github.com/ckan/ckan/assets/56607329/76f1b724-fb8c-4bea-bba7-fe113f48d3f4)

**Reply by FWICSS** [reply]
![CleanShot 2024-06-11 at 11 07 43@2x](https://github.com/ckan/ckan/assets/56607329/d5192be3-955c-49d7-91d7-7516bfe684af)

**Comment by tino097** [comment]
@FWICSS we have released a new version of Datapuhser Plus that works like [extension](https://github.com/dathere/datapusher-plus/releases/tag/1.0.1) and you dont need to start it as a separate service

**Reply by FWICSS** [reply]
is there any documentation of the installation like an extension?
